apps/profile-api/src/services/organisations/translations.ts

```diff
@@ -0,0 +1,36 @@
+import en from "./en.json" with { type: "json" };
+import ga from "./ga.json" with { type: "json" };
+
+export const translations = {
+  en,
+  ga,
+};
+
+export type Translations = typeof translations;
+
+export const getTranslation = (
+  key: keyof Translations[keyof Translations],
+  lang: keyof Translations,
+) => {
+  return translations[lang][key];
+};
+
+export const getOrganisationTranslation = (organisationId: string) => {
+  if (!Object.keys(translations.en).includes(organisationId)) {
+    throw new Error(`Organisation ${organisationId} not found`);
+  }
+
+  const key = organisationId as keyof Translations[keyof Translations];
+
+  return {
+    id: organisationId,
+    translations: {
+      en: {
+        name: translations.en[key],
+      },
+      ga: {
+        name: translations.ga[key],
```

Comment:
```suggestion
        name: translations.ga[key] ?? translations.en[key],
```

So to ensure we always have a value